### ISSUE TITLE
How to set WebGL Metadata from the Kameleo API?

### ISSUE THREAD

**Issue by JonathanR25** [kameleo-io/local-api-client-typescript#7]
What would be the proper code syntax form to set the WebGL Metadata from the Kameleo API?

Also, could your provide some example code?

**Comment by MBailey01** [comment]
Here is an example that you can use to set the WebGL Metadata through the API:

```
const createProfileRequest = BuilderForCreateProfile
  .forBaseProfile(chromeBaseProfileList[0].id)
  .setRecommendedDefaults()
  .setWebglMeta(
    'manual',
    {
      vendor: 'Google Inc. (AMD)',
      renderer: 'ANGLE (AMD Radeaon (TM) RX 560 Direct3D11 vs_5_0),
    },
  )
  .build();
const profile = await client.createProfile({ body: createProfileRequest });
```

These answers are provided from the Kameleo [Set WebGL Metadata from API](https://help.kameleo.io/hc/en-us/articles/13381644742300-Set-WebGL-Metadata-from-API) article.

**Comment by KameleoStaff** [comment]
We recently released Kameleo v4.0.0, which introduced several changes and improvements to the platform. As a result, this code has been updated to be fully compatible with the latest version.

```
@type {import('@kameleo/local-api-client').CreateProfileRequest} */
const createProfileRequest = {
  fingerprintId: fingerprints[0].id,
  webglMeta: {
    value: 'manual',
    extra: {
      vendor: 'Google Inc. (AMD)',
      renderer: 'ANGLE (AMD Radeon (TM) RX 560 Direct3D11 vs_5_0)',
    },
  },
};

const profile = await client.profile.createProfile(createProfileRequest);
```

These answers are provided from the Kameleo [Set WebGL Metadata from API](https://help.kameleo.io/hc/en-us/articles/13381644742300-Set-WebGL-Metadata-from-API) article.
